sqlglot/generator.py

```diff
@@ -53,6 +53,8 @@ class Generator:
         exp.TsOrDsAdd: lambda self, e: f"TS_OR_DS_ADD({self.sql(e, 'this')}, {self.sql(e, 'expression')}, {self.sql(e, 'unit')})",
     }
 
+    NULL_ORDERING_SUPPORTED = True
```

Comment:
update the doc string with the enums in both this and the parser

Reply:
oh this is no longer an enum?

Reply:
So this allows a dialect to set their generator to if they support null ordering or not. This is so MySQL and Starrocks can say they don't have null ordering.

Reply:
nvm, this is something else,

can you add null_ordering to the doc strings